leaderBoard/data2.json

```diff
@@ -0,0 +1 @@
+[11,0,0,1,2,0,0,3,1,0,0,1,0,0,0,1,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,1,0,1,1,0,0,0,0,1,0,0,1,0,0,0,0,0,0]
```

Comment:
What data is this?

Reply:
the solutions data fetched per user, was saved in an array.